test/integration/storage/test_objects.rb

```diff
@@ -0,0 +1,114 @@
+require "helpers/integration_test_helper"
+
+class TestObjects < FogIntegrationTest
+  def setup
+    # Uncomment this if you want to make real requests to GCE (you _will_ be billed!)
```

Comment:
Ok, so, my understanding is people will not generally want to go and edit tests to run them.
AFAIK - there's no mocking in place for the new set of integration tests, so this will fail anyways.

IMO as soon as they are actually functional, the warning/config option should be on a very high level. For example, when you run `rake test` with no arguments it throws a warning or asks you whether you want to continue with an option for mocks somewhere in top layer test config. + having a default yes option for CI, like `rake test:bill_me_money` or something. What do you think?